client/my-sites/email/email-providers-comparison/style.scss

```diff
@@ -183,6 +183,15 @@
 					margin-top: 0;
 				}
 			}
+
+			.email-providers-comparison__titan-mailbox-action-continue {
+				width: 100%;
```

Comment:
Could you explain why this `width` is needed? I believe the button should automatically be full-width when the wrapping container is using `flex-direction: column;`. So I think we can remove this attribute as well as the `width: auto;` override on L191.

Reply:
You are right Dale, I saw strange behavior that required this bit. Although that strange behavior is not replicable now 😖 . I've removed that CSS adjustment.